ci/buildkite-pipeline.sh

```diff
@@ -139,7 +139,14 @@ wait_step() {
 }
 
 all_test_steps() {
-  command_step checks ". ci/rust-version.sh; ci/docker-run.sh \$\$rust_nightly_docker_image ci/test-checks.sh" 20
```

Comment:
might be a little cleaner to add queue a fourth optional arg to `command_step_checks`? something like

```
local queue="${4:-solana}"
```

and replace the hardcoded queue accordingly

Reply:
fixed 🪖